File: matchmaker/server/src/main/java/ru/ifmo/se/dating/matchmaker/api/HttpPeopleApi.java
package ru.ifmo.se.dating.matchmaker.api;

import java.util.Set;

import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Service;

import ru.ifmo.se.dating.matchmaker.model.generated.AttitudeKind;
import ru.ifmo.se.dating.matchmaker.api.generated.PeopleApiDelegate;

@Service
class HttpPeopleApi implements PeopleApiDelegate {

    @Override
    public ResponseEntity<Void> peoplePersonIdAttitudesIncomingAttitudeKindPost(
            Long personId, AttitudeKind attitudeKind) {
        return ResponseEntityStub.create();
    }

    @Override
    public ResponseEntity<Set<Long>> peoplePersonIdMatchesGet(Long personId) {
        return ResponseEntityStub.create();
    }
}

File: matchmaker/server/src/main/kotlin/ru/ifmo/se/dating/matchmaker/api/HttpPeopleApi.kt
package ru.ifmo.se.dating.matchmaker.api

import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Service
import ru.ifmo.se.dating.matchmaker.api.generated.PeopleApiDelegate
import ru.ifmo.se.dating.matchmaker.model.generated.AttitudeKind

@Service
internal class HttpPeopleApi : PeopleApiDelegate {
    override fun peoplePersonIdAttitudesIncomingAttitudeKindPost(
        personId: Long, attitudeKind: AttitudeKind
    ): ResponseEntity<Unit> =
        ResponseEntityStub.create()

    override fun peoplePersonIdMatchesGet(personId: Long): ResponseEntity<List<Long>> =
        ResponseEntityStub.create()
}
